src/background/target-page-controller.ts

```diff
@@ -93,7 +95,7 @@ export class TargetPageController {
             });
 
             for (const activeTab of activeTabs) {
-                this.sendTabVisibilityChangeAction(
+                await this.sendTabVisibilityChangeAction(
```

Comment:
Can this use `Promise.all` instead of awaiting in sequence? (ideally across all windows, too, not just across tabs per-window)